build/Dockerfile

```diff
@@ -9,7 +9,7 @@ ARG OC_CHANNEL=stable
 
 
 RUN dnf -y install https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm &&\
-    dnf install -y python38 jq git unzip chromium chromedriver redhat-lsb-core &&\
+    dnf install -y python38 jq git unzip chromium chromedriver redhat-lsb-core s3cmd python3-virtualenv &&\
```

Comment:
Yes. Without that change, "venv" was used. But venv cannot create a virtualenv with a different version of Python. ods-ci uses Python 3.7+ Python features but ubi8/streams8 has 3.6 by default.

Reply:
I might missing something here, but I still don't see the need of using virtualenv, (or venv before the change), because when you use a container you can install the Python version of your choice directly using the package manager. and will still be isolated Could you please elaborate this a bit more?